build.sbt

```diff
@@ -23,7 +23,8 @@ libraryDependencies ++= Seq(
   minitest % "test",
   minitestLaws % "test",
   algebird % "test",
-  chill % "test"
+  chill % "test",
+  avro4s % "test"
```

Comment:
Just making sure that no dependencies tagged as `test` are published the central repo because of this line in `build.sbt` `publishArtifact in Test := false` ?

Reply:
What do u think we should do ? Should we publish them in the repo ? This issue https://github.com/lightbend/kafka-streams-scala/issues/56 is still open and it asks for this.

Reply:
No, I don't think it makes sense to publish them.  I think directing people from the README to the tests path in the repository is good enough.  I just wanted to confirm that these deps don't get sucked up and pollute downstream projects.

Reply:
I also thought so .. @seglo - could u pls comment on that issue #56 where the person is asking for this.